core/system/src/main.c

```diff
@@ -50,6 +51,11 @@ UVISOR_NOINLINE void uvisor_init_post(void)
         /* init MPU */
         vmpu_init_post();
 
+        /* init page allocator */
+        page_allocator_init(__uvisor_config.page_start,
```

Comment:
Does the initialization need to run after `vmpu_init_post`? Is there a vMPU-specific dependency? If so please add a comment.